utilities/generate_api.py

```diff
@@ -138,6 +167,8 @@ def main():
                         help='Don\'t print informational instructions')
     args = parser.parse_args()
 
+    dump_versions(args.config_file)
```

Comment:
Can you put this under `if not args.quiet`? Otherwise, it will print the same thing over and over when running from `batch_generate_apis.py`.

Reply:
I think something needs to be printed for batch_generate_apis.py, but I'm not sure the best way to share the functionality. I think repeatedly printing it is not ideal, but is better then nothing

Reply:
Call this function (`dump_versions`) directly from `batch_generate_apis.py`.

Reply:
That works